src/config/toml_writer.c

```diff
@@ -41,15 +41,17 @@ bool writeFTLtoml(const bool verbose)
 		log_info("Writing config file");
 
 	// Write header
-	fputs("# This file is managed by pihole-FTL\n#\n", fp);
-	fputs("# Do not edit the file while FTL is\n", fp);
```

Comment:
Why did you remove this and the following line? 
I'm not sure if users understand that this is meant with the new "This file is managed by pihole-FTL"

Reply:
Because it is explicitly supported by now. FTL actively monitors the file and reloads it on any changes. This is also how `pihole-FTL --config ...` works behind the scenes: It modifies the file on disk and the currently running FTL instance sees that and picks up the changes.